.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: build
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: centos-stream-8
+            shortcut: cs8
+            container-name: el8stream
+          - name: centos-stream-9
+            shortcut: cs9
+            container-name: el9stream
+    name: ${{ matrix.name }}
+    container:
+      image: quay.io/ovirt/buildcontainer:${{ matrix.container-name }}
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: automation/check-patch.sh
+      # TODO: Split to separate steps?
+
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v2
```

Comment:
Don't we need to use `Upload RPMs the oVirt way` -  https://github.com/oVirt/upload-rpms-action?

Reply:
the only thing is now it also includes tar.gz. it's overall very small so i don't mind keeping it in.